extension/chrome/elements/pgp_block.ts

```diff
@@ -378,8 +378,10 @@ Catch.try(async () => {
         isHtml = true;
       } else if (typeof decoded.text !== 'undefined') {
         decryptedContent = decoded.text;
+      } else {
+        decryptedContent = '';
```

Comment:
This takes care of setting the (empty), so that the whole mime message does not get wrongly rendered as abefore.